src/main/java/proto/objects.proto

```diff
@@ -0,0 +1,30 @@
+/*
+Instead of encoding all data in one proto, we will instead preface the proto
```

Comment:
For the multi-line comments in header, maybe all use //, or /* and every line start with a *

I'd suggest  to use unified comment style.